### PR TITLE
LPS-189843 Check update permission for showing convert actions

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/helper/LayoutActionsHelper.java [hunk 1]
```diff
@@ -88,8 +88,12 @@ public boolean isShowConfigureAction(Layout layout) throws PortalException {
 			_themeDisplay.getPermissionChecker(), layout);
 	}
 
-	public boolean isShowConvertLayoutAction(Layout layout) {
-		if (_isLiveGroup() ||
+	public boolean isShowConvertLayoutAction(Layout layout)
+		throws PortalException {
+
+		if (LayoutPermissionUtil.containsLayoutUpdatePermission(
```
Comment (ruben-pulido) [inline modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/helper/LayoutActionsHelper.java:94]: Should this be negated?

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/137552

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**